evmrpc/rpcstack_test.go

```diff
@@ -614,3 +614,28 @@ func (s *testService) Greet() string {
 func (s *testService) Sleep() {
 	time.Sleep(1500 * time.Millisecond)
 }
+
+func TestHttpDenyList(t *testing.T) {
+	const (
+		expectRes = `{"jsonrpc":"2.0","id":null,"error":{"code":-32601,"message":"the method test_sleep does not exist/is not available"}}`
+	)
+	// Set-up server
+	timeouts := rpc.DefaultHTTPTimeouts
+	timeouts.WriteTimeout = time.Second
+	srv := createAndStartServer(t, &evmrpc.HTTPConfig{
+		DenyList: []string{"test_sleep"},
```

Comment:
I don't think the `test_sleep` endpoint exists on this server. Might be good to try to denylist an endpoint that exists like `test`.

Reply:
oh just saw this, I think we added that as a new module, if you look at previous test cases, it seems to be a valid endpoint